src/components/Footer.jsx

```diff
@@ -1,7 +1,46 @@
+import img1 from "@/public/footer.webp";
+import wavetop from "@/public/waveTop.webp";
+import { footerLinks } from "../data/footerLinks";
+import Link from "next/link";
+import Image from "next/image";
+
 const Footer = () => {
   return (
-    <div className="">
-      <p>Footer</p>
+    <div className="flex flex-col relative w-page h-auto">
+      <div className="flex w-screen mt-28">
+        <Image alt="img1" src={img1} className="w-screen h-full" />
+      </div>
```

Comment:
```suggestion
        <Image alt="img1" src={img1} className="flex w-screen h-full mt-28" />
```

combine classnames.
do the same for wavetop and the UCR CLUB SWIM link.